app/service/group_service.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+from typing import Any
+from sqlalchemy.orm import Session
+from app.repository.group_repository import GroupRepository
+
+
+class GroupType(str, Enum):
```

Comment:
This seems to be either a constant or a model?

Reply:
Can you please try to remove the `.idea` folder as well?

Reply:
this I defined to be a constant